api/src/main/java/org/openmrs/module/patientmatching/PatientMatchingActivator.java

```diff
@@ -72,7 +72,7 @@
 	
 	private static final String extention = "patientMatching";
 	
-	public final static String CONFIG_FILE = "link_config.xml";
```

Comment:
Could you be knowing why it was originally prefixed with `link_`?

Reply:
Arbitrary I guess, the link_ prefix is not intuitive, I rather have it as matchingConfig.xml or patientMatchingConfig.xml just to be different from a module config.xml file

Reply:
Renamed it back